strings-csharp/StringFormattableString/StringFormattableStringTests/FormattableStringsUnitTests.cs

```diff
@@ -0,0 +1,53 @@
+using StringFormattableString;
+using System.Runtime.CompilerServices;
+
+namespace StringFormattableStringTests
+{
+    [TestClass]
+    public class FormattableStringsUnitTests
+    {
+        private readonly FormattableStringsMethods _formattableStringMethods;
+
+        public FormattableStringsUnitTests()
+        {
+            _formattableStringMethods = new FormattableStringsMethods();
+        }
+
+        [TestMethod]
+        public void GivenAStringObject_WhenStringOperationsApplied_VerifyAccurateResults()
+        {
+            var stringObj = "Jane";
+            var sampleString = _formattableStringMethods.StringExample();
+
+            Assert.AreEqual(stringObj.Count(), stringObj.Length);
+            Assert.IsFalse(stringObj.Equals(sampleString));
+            Assert.AreEqual(sampleString.CompareTo(stringObj), 1);
+            Assert.IsInstanceOfType(sampleString, typeof(string));
+        }
+
+        [TestMethod]
+        public void GivenAFormattableStringObject_WhenFormattableStringOperationsApplied_VerifyAccurateResults()
+        {
+            var sampleFormattableString = _formattableStringMethods.FormattableStringExample();
+
+            Assert.AreEqual("John", sampleFormattableString.GetArgument(0));
+            Assert.AreEqual(sampleFormattableString.ArgumentCount, 2);
+            Assert.IsInstanceOfType(sampleFormattableString, typeof(FormattableString));
+            Assert.IsInstanceOfType(sampleFormattableString.ToString(), typeof(string));
```

Comment:
This `Assert` isn't necessary as it is in essence testing the framework. By definition calling `ToString` returns a string.